tests/postgres_tests/test_json.py

```diff
@@ -386,6 +386,9 @@ def test_key_sql_injection(self):
             queries[0]['sql'],
         )
 
+    def test_field_contains(self):
```

Comment:
I'd move this test under the existing `test_contains` method and name it `test_nested_contains`.

Reply:
Thanks, I did that. I wasn't sure if there was a rule to always add tests at the end of the `TestCase`